cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -905,6 +905,12 @@ spec:
             - name: ADVANCED_NETWORK_STATS
               value: "false"
             {{- end}}
+            {{- end }}
+            {{- end }}
+            {{- end }}
+            {{- if .Values.kubecostModel }}
+            {{- if .Values.kubecostModel.collections }}
+            {{- if .Values.kubecostModel.collections.cache }}
```

Comment:
```suggestion
            {{- if (((.Values.kubecostModel).collections).cache) }}
```

Reply:
Seems to work without the parens. Are they required? Just following existing patterns.

Reply:
Yes it works but this is a more efficient pattern. Makes it simpler.

Reply:
Added the parens, still works.

Reply:
> Yes it works but this is a more efficient pattern. Makes it simpler.

I can say with 100% certainty that this is not conventional go templating and there is 0 added efficiency from adding parens like this. There's no ambiguous access anywhere in this chain.